src/components/InpageNav/InpageNav.stories.tsx

```diff
@@ -30,7 +30,7 @@ const InpageNavExample = () => {
   return <InpageNav isMinimal={isMinimal} links={links} />;
 };
 
-const stories = storiesOf('Components|Page header', module);
+const stories = storiesOf('Components|InPageNav', module);
```

Comment:
This was grouped together with the Page header modules for easy author reference

Reply:
Reverted.